src/control/cmd/dmg/main.go

```diff
@@ -262,12 +262,14 @@ and access control settings, along with system wide operations.`
 			// to the specified file.
 			log = log.
 				WithErrorLogger(logging.NewErrorLogger("dmg", f)).
+				WithNoticeLogger(logging.NewNoticeLogger("dmg", f)).
 				WithInfoLogger(logging.NewInfoLogger("dmg", f)).
-				WithDebugLogger(logging.NewDebugLogger(f))
+				WithDebugLogger(logging.NewDebugLogger(f)).
+				WithTraceLogger(logging.NewTraceLogger(f))
 		}
 
 		if opts.Debug {
-			log.WithLogLevel(logging.LogLevelDebug)
+			log.SetLevel(logging.LogLevelTrace)
```

Comment:
Probably a stupid question, could we still choose the LogLevelDebug when calling the dmg command ?

Reply:
I could be convinced otherwise, but my thought was that if the admin has been asked to run any of the utilities with the `--debug` flag, it is because the developers are trying to gather as much information as possible, and therefore we are OK with the extra verbosity of TRACE logging in the control plane components. This is separate from setting the log level in a config file, as it may be appropriate/useful to run with DEBUG during system validation, but TRACE is too verbose to leave enabled except for short periods of time.